package.json

```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "prop-types": "^15.7.2",
-    "react-script-hook": "^1.0.15"
+    "tr-tmp-react-script-hook": "1.0.17"
```

Comment:
Since this module name is changing here, does it also need to be updated where it's being imported?

Reply:
Consistency is a wonderful thing.